src/main/resources/db/migration/V1_60__insert_judicial_location_mapping_table.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO judicial_location_mapping (epimms_id,judicial_base_location_id,building_location_name,base_location_name,service_code) VALUES
```

Comment:
this sql does not have schema, pls add schema.

Reply:
the previous scripts which aren't related to elinks don't have a schema specified, so they will use the default one used in the connection string to the db which is dbjuddata